lecli/api_utils.py

```diff
@@ -183,12 +182,12 @@ def get_account_resource_id():
 
 def get_named_logkey_group(name):
     """
-    Get named log-key group from the config file.
+    Get named log-key list from the config file.
 
-    :param name: name of the group
+    :param name: name of the log key list
     """
 
-    section = 'LogGroups'
+    section = 'CLI_Favorites'
```

Comment:
Does this mean, users will need to update their configuration files?

Reply:
yep - having the existing section LogGroup still in the config will not cause problems as it gets ignored - but they will need to add the new section for CLI_Favorites. On a fresh install the default config will have the new section and not the old. @stopal-r7 mentioned we will not be supporting backward compatibility on this.

Reply:
I wonder if we can include some code to automatically migration users' configuration files for them. Should be simple enough, right?

Reply:
Yeah either rename if section found. OR read from both sections (CLI_favorites and LogGroups) as format is unchanged.

Reply:
That's a change that'll need to be discussed with Product - I have not discussed the role out of these changes as Safa seemed pretty sure of what was to be done, but I'l follow up on it.

Reply:
hhmmm If we are going to handle existing sections, I'm thinking maybe going with renaming the section as the original idea of this work is to move away from use of the terminology of LogGroups?

Reply:
Ok with me.